### PR TITLE
Improves in-game immersion

### PR DESCRIPTION
**Pull request by Spess-Rando**
:cl: Slignerd
add: All personnel have been given a lecture on Galactic Basic to discourage them from using netspeak in real life.
/:cl:

### CODE REVIEW

**Comment by Buggy123** [comment]
*breaths in*
*blinks*
Hi guys I just
*breaths out*
*breaths in*
*shifts weight*
was wondering if
*breaths out*
*breaths in*
our speech is
*breaths out*
*breaths in*
*swallows saliva*
realistic enough or
*breaths out*
*breaths in*
*shifts weight*
not.

**Comment by Qustinnus** [comment]
k.ek

**Comment by JJRcop** [comment]
Should be soft-coded in strings

**Comment by Repukan** [comment]
Smh tbh fam

**Comment by factoryman942** [comment]
stops people from saying s you c c
👎

**Comment by Slignerd** [comment]
So, it looks like it just doesn't work at all.

**Comment by Slignerd** [comment]
So, it looks like it just doesn't work at all.

**Comment by CameronWoof** [comment]
I thought Goof was the only person allowed to make meme PRs that don't even work.

**Comment by DaxDupont** [comment]
>ok gets replaced by okay

File: code/game/say.dm [hunk 1]
```diff
@@ -26,6 +26,27 @@ GLOBAL_LIST_INIT(freqtospan, list(
...
+	message = replacetext(message, " btw ", " by the way ")
+	message = replacetext(message, " thx ", " thanks ")
+	message = replacetext(message, " ty ", " thanks ")
+	message = replacetext(message, " wtf ", " what the fuck ")
+	message = replacetext(message, " afaik ", " as far as I know ")
+	message = replacetext(message, " etc ", " et cetera ")
+	message = replacetext(message, " y ", " why ")
+	message = replacetext(message, " u ", " you ")
+	message = replacetext(message, " k ", " 'kay ")
+	message = replacetext(message, " ok ", " okay ")
```
Comment (DaxDupont) [inline code/game/say.dm]: What the fuck

**Review by DaxDupont — CHANGES_REQUESTED** [review]

File: code/game/say.dm (hunk 1 above)
Comment (DaxDupont) [reply code/game/say.dm]: delete this

Comment (JJRcop) [reply code/game/say.dm]: yuk

**Comment by Slignerd** [comment]
It doesn't work, and nobody seem,s to like it anyway, so I guess there's no use trying to fix it.

**Comment by nero1024** [comment]
I've got a better idea for emoticons, why not turn them into emotes?
Example:
:) = me"smiles"
:^) = me"smiles like a smartass"
owo = me"eyes widen with wonder"

**Comment by Slignerd** [comment]
That does sound neat, though a bit harder to implement.

File: code/game/say.dm [hunk 2]
```diff
@@ -26,6 +26,27 @@ GLOBAL_LIST_INIT(freqtospan, list(
...
+	message = replacetext(message, " lol ", " hahahah ")
+	message = replacetext(message, " kek ", " hahahah ")
```
Comment (kevinz000) [inline code/game/say.dm]: :(

**Comment by kevinz000** [comment]
kek

**Comment by sawrge** [comment]
cool idea if u can get it working

**Review by PKPenguin321 — COMMENTED** [review]
glad this is closed in its current state but i think just the emojis would be worthy of a different PR

File: code/game/say.dm [hunk 3]
```diff
@@ -26,6 +26,27 @@ GLOBAL_LIST_INIT(freqtospan, list(
...
+	message = replacetext(message, " lmao ", " hahahah ")
```
Comment (XDTM) [inline code/game/say.dm]: You can replace these with a laugh emote

**Comment by XDTM** [comment]
Wait it's closed nvm

File: code/game/say.dm [hunk 4]
```diff
@@ -26,6 +26,27 @@ GLOBAL_LIST_INIT(freqtospan, list(
 	var/list/spans = get_spans()
 	if(!language)
 		language = get_default_language()
+	if(findtext(message, "((") || findtext(message, "))"))
+		to_chat(src, "<span class='userdanger'>Do not use IC chat for OOC messages.</span>")
```
[the remark this answers is not in the document]
Comment (Slignerd) [reply code/game/say.dm]: You can use OOC vocabulary in IC to help people, but you really should not use (( and )) in the process.

**Pull request closed without merging**